lib/mayaUsd/base/mayaUsd.h.src

```diff
@@ -36,11 +36,10 @@ namespace MAYAUSD_VERSIONED_NS {}
 namespace MAYAUSD_NS {
     using namespace MAYAUSD_VERSIONED_NS;
 }
- 
-// Macros to place the MayaUsd symbols in the versioned namespace, which is how
-// they will appear in the shared library, e.g. MayaUsd_v1::Class.
+
+// Deprecated macro (because it causes problems with clang-format)
```

Comment:
This comment is no longer accurate, right?  We should restore the previous comment, perhaps fixing it to change the plural "Macros" to "Macro".